salt/utils/sanitisers.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
```

Comment:
@thatch45 I see what you did here. :laughing: Although, renamed to American spelling.